src/plonkish/backend/halo2.rs

```diff
@@ -804,3 +884,38 @@ pub(crate) fn chiquito2Halo2<F: PrimeField + From<u64> + Hash>(
 ) -> ChiquitoHalo2<F> {
     ChiquitoHalo2::new(circuit)
 }
+
+impl<F> ExpressionWithColumn for Expression<F> {
+    fn involves(&self, column: &cColumn) -> bool {
```

Comment:
@alxkzmn why do we need this method in both types of expressions?

Reply:
Because of the imported expressions ?

Reply:
Yes, it is necessary because we have `PolyExpr::Halo2Expr` here: https://github.com/privacy-scaling-explorations/chiquito/blob/4b077b2b598c42d2cd5e2c5dc1ec5b79c30c96f5/src/plonkish/ir/mod.rs#L194

Reply:
Ok